frontend/packages/dev-console/src/components/helm/HelmReleaseDetails.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import { Firehose } from '@console/internal/components/utils';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import { SecretModel } from '@console/internal/models';
+import HelmReleaseDetailsPage from './HelmReleaseDetailsPage';
+
+export interface HelmReleaseDetailsProps {
+  match: RMatch<{
+    ns?: string;
+    name?: string;
+  }>;
+}
+
+const HelmReleaseDetails: React.FC<HelmReleaseDetailsProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  const helmReleaseName = match.params.name;
+  return (
+    <>
+      <NamespaceBar />
+      <Firehose
+        resources={[
+          {
+            kind: SecretModel.kind,
+            namespace,
+            isList: true,
```

Comment:
Does this need to be a list? You're only looking for a single resource.

Reply:
Yes I am looking for a single secret resource but it doesn't seem to work if isList is not true

Reply:
What doesn't work? You get no data back?

Reply:
if isList is false then it gives a list of all the secrets under the namespace instead of the particular secret resource with the name label.